contracts/RewardsLiquidator.vy

```diff
@@ -147,98 +239,156 @@ def _get_eth_anchor_price() -> uint256:
     answered_in_round: uint256 = 0
 
     (round_id, answer, started_at, updated_at, answered_in_round) = \
-        ChainlinkAggregatorV3Interface(CHAINLINK_ETH_USD_FEED).latestRoundData()
+        ChainlinkAggregatorV3Interface(chainlink_price_feed).latestRoundData()
 
     assert updated_at != 0
+    if reverse:
+        return  (10 ** (18 + price_decimals)) / convert(answer, uint256)
+    return convert(answer, uint256) * (10 ** (18 - price_decimals))
```

Comment:
Could you please put this `return` inside an `else` block for readability?

Reply:
is no longer relevant, since I removed `if` statement